docs/agents/spec-process.md

```diff
@@ -0,0 +1,69 @@
+Every change that affects more than one agent should be initiated via a change to the specification rather than creating a cross-agent issue.
+
+## Process
+
+1. Agents discussion issue \
+  If there's a lot of uncertainty about what the proposal is,
+  a discussion issue should be created.
+  In doubt, lean towards not creating a discussion issue and start with a draft PR.
+  Discussion issues should not have votes/agent checkboxes.
+  It's not expected that all agents participate in the initial discussions, although everyone is invited to do so.
+1. Draft PR \
+  It's fine if the first draft of the spec change even if details are unclear and need discussion.
+  The PR should include which agents are affected by this change.
+  Specifically, it should always be clear whether the RUM agent is affected by this.
+  This doesn't have to be determined yet when creating the draft PR,
+  but it is required before marking the PR as `Ready for Review`.
```

Comment:
any other suggestions for what makes a good spec change?  Some ideas:
* minimum version of the elastic stack required for this, in case of new intake. 
* link to related meta issues

Reply:
I think that part is TBD. It's certainly an important part of the spec but I wonder if we could define that in a follow-up. I think after going through a couple of iterations of this process we can give better guidance.

> minimum version of the elastic stack required for this, in case of new intake.

Wouldn't that always be the version the spec PR is assigned to? If not, would you add that to the PR description or in the agent spec itself?

Reply:
Tackling this in a follow up sounds fine as does capturing version information in the milestone.  We always have the option to add that later if things are unclear.